core/bin/check_dsp

```diff
@@ -62,7 +62,7 @@ foreach my $dir (@DIRS)
   my @file_list = grep {! m/^(\..*|CVS|\.svn)$/} readdir(BASEDIR);
   closedir(BASEDIR);
   my @sd = ();
-  if    ($dir =~ m!\bvcl$!)   { @sd = qw(vcl_sys iso generic gcc egcs gcc-295 gcc-libstdcxx-v3 sgi sunpro win32-vc60 win32-vc70 win32-vc8 emulation borland55 borland56 mwerks stlport internal); }
+  if    ($dir =~ m!\bvcl$!)   { @sd = qw(vcl_sys iso generic gcc egcs gcc-295 gcc-libstdcxx-v3 sgi sunpro win32-vc60 win32-vc70 win32-vc8 emulation stlport internal); }
```

Comment:
Good enough ... wasn't going to hold it up anyway, just wanted it on record
:)

On Thu, Jan 14, 2016 at 7:15 AM, Hans Johnson <notifications@github.com>
wrote:

> In core/bin/check_dsp
> <https://github.com/vxl/vxl/pull/105#discussion_r49718959>:
>
> > @@ -62,7 +62,7 @@ foreach my $dir (@DIRS)
> >    my @file_list = grep {! m/^(\..*|CVS|\.svn)$/} readdir(BASEDIR);
> >    closedir(BASEDIR);
> >    my @sd = ();
> > -  if    ($dir =~ m!\bvcl$!)   { @sd = qw(vcl_sys iso generic gcc egcs gcc-295 gcc-libstdcxx-v3 sgi sunpro win32-vc60 win32-vc70 win32-vc8 emulation borland55 borland56 mwerks stlport internal); }
> > +  if    ($dir =~ m!\bvcl$!)   { @sd = qw(vcl_sys iso generic gcc egcs gcc-295 gcc-libstdcxx-v3 sgi sunpro win32-vc60 win32-vc70 win32-vc8 emulation stlport internal); }
>
> Yes. I already have those patches created. Just letting them trickle in
> one at a time. There is some inter-relations between these compilers, so
> identifying completely clean patch sets has been difficult.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/vxl/vxl/pull/105/files#r49718959>.
>



-- 
David Stoup
R&D Engineer

Kitware, Inc.
28 Corporate Drive
Clifton Park, NY. 12065
518-881-4949 (W)
518-312-3946 (M)
518-371-4573 (F)